pages/form-builder/support/[[...supportType]].tsx

```diff
@@ -33,400 +34,457 @@ export default function Contactus() {
   const [isSuccessScreen, setIsSuccessScreen] = useState(false);
   const [isSubmitting, setIsSubmitting] = useState(false);
   const [errorMessage, setErrorMessage] = useState("");
+  const headingSuccessRef = useRef(null);
 
-  const handleRequest = async (
-    name: string,
-    email: string,
-    department: string,
-    branch: string,
-    jobTitle: string,
-    request: [string] | string,
-    description: string
-  ) => {
-    const token: string = (await getCsrfToken()) || "";
-    return axios({
-      url: "/api/request/support",
-      method: "POST",
-      headers: {
-        "Content-Type": "application/json",
-        "X-CSRF-Token": token,
-      },
-      data: { supportType, name, email, department, branch, jobTitle, request, description },
-      // If development mode disable timeout
-      timeout: process.env.NODE_ENV === "production" ? 60000 : 0,
-    }).catch((err) => {
-      logMessage.error(err);
-      setErrorMessage(t("contactus.errors.submissionError"));
-    });
-  };
+  useFocusIt({ elRef: headingSuccessRef, dependencies: [isSuccessScreen] });
```

Comment:
Since we have an easy way to focus a header I went with that vs. using a live region around an entire page update. At the time it was a quick solution, technically correct, but the header focus to indicate a new page loading is cleaner